src/wallet/wallet2.cpp

```diff
@@ -2385,6 +2365,10 @@ bool wallet2::store_keys(const std::string& keys_file_name, const epee::wipeable
   }
 
   rapidjson::Value value2(rapidjson::kNumberType);
+
+  value2.SetInt(m_key_on_device?1:0);
+  json.AddMember("key_on_device", value2, json.GetAllocator());
```

Comment:
Should be added to the ctor init list.

Reply:
Can you explain? I do not see what to do.

Reply:
Add "m_key_on_device(false)" to the wallet2 constructor init list, near the top of the file. Safety to ensure it's always initialized.

Reply:
done